internal/execute/tsc.go

```diff
@@ -49,7 +49,7 @@ func executeCommandLineWorker(sys System, cb cbType, commandLine *tsoptions.Pars
 		}
 
 		fileOrDirectory := tspath.NormalizePath(commandLine.CompilerOptions().Project)
-		if fileOrDirectory != "" || sys.FS().DirectoryExists(fileOrDirectory) {
+		if sys.FS().DirectoryExists(fileOrDirectory) {
```

Comment:
This line was ported incorrectly 
https://github.com/microsoft/TypeScript/blob/0aac72020ee8414218273f654eb7ce1dc2dd0d6b/src/compiler/executeCommandLine.ts#L603